geode-web/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ConnectCommandWithSSLMultiKeyTest.java

```diff
@@ -0,0 +1,199 @@
+import static org.apache.geode.distributed.ConfigurationProperties.SSL_TRUSTSTORE_PASSWORD;
+import static org.apache.geode.distributed.ConfigurationProperties.SSL_TRUSTSTORE_TYPE;
+import static org.apache.geode.test.util.ResourceUtils.createTempFileFromResource;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.util.Properties;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.TemporaryFolder;
+
+import org.apache.geode.security.SecurableCommunicationChannels;
+import org.apache.geode.test.dunit.IgnoredException;
+import org.apache.geode.test.dunit.rules.ClusterStartupRule;
+import org.apache.geode.test.dunit.rules.MemberVM;
+import org.apache.geode.test.junit.categories.GfshTest;
+import org.apache.geode.test.junit.rules.GfshCommandRule;
+
+@Category(GfshTest.class)
+public class ConnectCommandWithSSLMultiKeyTest {
+  private static MemberVM locator;
+  private static Properties sslProperties;
+  private static Properties gfshSslProperties;
+  private OutputStream out = null;
+  private File sslConfigFile = null;
+
+  @Rule
+  public GfshCommandRule gfsh = new GfshCommandRule();
+
+  @Rule
+  public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  @ClassRule
+  public static ClusterStartupRule clusterRule = new ClusterStartupRule();
+
+  @BeforeClass
+  public static void beforeClass() {
+    // The files were generated using the following commands.
+    // The final step import the certificates in the "wrong" order (SSL implementation uses by
+    // default the first one found, ignoring the alias).
+    //
+    // openssl genrsa -out ca.key 2048
+    // # It will be necessary to provide a keystore password and the CN for each.
+    // # Use "client" for the CN of the client keystore and "server" for the CN of the server key.
+    // keytool -genkeypair -alias client -keyalg RSA -keystore client.jks
+    // keytool -genkeypair -alias server -keyalg RSA -keystore server.jks
+    // # Export CSRs
+    // keytool -certreq -alias client -keystore client.jks -file client.csr
+    // keytool -certreq -alias server -keystore server.jks -file server.csr
+    // # Create a self-signed cert for the CA
+    // openssl req -new -x509 -key ca.key -out ca.crt
+    // # Sign them
+    // openssl x509 -req -in client.csr -CA ca.crt -CAkey ca.key -out client.crt -CAcreateserial
+    // openssl x509 -req -in server.csr -CA ca.crt -CAkey ca.key -out server.crt -CAcreateserial
+    // # Import both of these leaf certs into a trust store but don't import CA
+    // keytool -importcert -keystore trusted.jks -file client.crt -alias client
+    // keytool -importcert -keystore trusted.jks -file server.crt -alias server
```

Comment:
I think this is confusing and not how certificates are typically handled. You should just need to import the `ca.crt` into the trust store and use that.

Reply:
Thanks @jdeppe-pivotal, 
I'll try to simplify the tests before merging the change.
Cheers.